### PR TITLE
LPS-193210

### PR DESCRIPTION
**Pull request by ces-quanhuynh**
Hi @locpham97 
Please help me to review this PR

# Motivation
[LPS-193210](https://liferay.atlassian.net/browse/LPS-193210) Add/Remove Feature or Release flag of LPS-185213

# Solution
LPS-193210 Adding ExtendedAttributeDefinition which having featureFlagKey = "LPS-185213" to CaptchaConfiguaration enableSimpleCaptchaHeadlessAPI. Thus, we can set value for this feature flag in portal-ext.properties to set enable/disable this configuration.

Thank you so much
cc @tototrinh @locpham97

### CODE REVIEW

**Comment by locpham97** [comment]
Please hide all the logics in the feature flags

File: modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java [hunk 1]
```diff
@@ -162,6 +164,10 @@ public String serveImageOutputStream(
 			HttpServletRequest httpServletRequest, OutputStream outputStream)
 		throws IOException {
 
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-185213")){
```
Comment (locpham97) [inline modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java:167]: no need in here

Comment (ces-quanhuynh) [reply modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java]: Removed it. Thanks

**Review by tototrinh — APPROVED** [review]
LGTM. Thanks.

**Review by locpham97 — APPROVED** [review]

**Pull request merged**